benchmark/text-generation/mistral_small.py

```diff
@@ -27,7 +25,7 @@ def main():
             export=True,
             batch_size=batch_size,
             sequence_length=seq_length,
-            auto_cast_type="fp16",
+            auto_cast_type="bf16",
```

Comment:
So it's now proven that bf16 is better?

Reply:
It actually does not make any difference as far as I can tell, and this seems to allow to load weights faster (because they are not converted).